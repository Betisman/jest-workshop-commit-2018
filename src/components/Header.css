
.container {
  display: flex;
  justify-content: center;
  align-items: center;
  height: 10vh;
  background-color: #373d49;
  color: #FFF;
}

.container .header {
  color: #FFF;
}