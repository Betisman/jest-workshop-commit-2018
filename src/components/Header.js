import React from 'react';
import style from './Header.css';

const Header = () => (
  <header className={style.container}>
    <h1 class={`ui header ${style.header}`}>H1</h1>
  </header>
);

export default Header;
