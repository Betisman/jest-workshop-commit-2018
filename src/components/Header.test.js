import React from 'react';
// helper library
import renderer from 'react-test-renderer';
import Header from './Header';

describe('Header', () => {
  it('<Header> should have the children content', () => {
    const tree = renderer.create((
      <Header>Create notes</Header>
    )).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
