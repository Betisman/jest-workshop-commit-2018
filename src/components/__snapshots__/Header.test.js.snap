// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header <Header> should have the children content 1`] = `
<header>
  <h1
    className="ui header undefined"
  >
    Create notes
  </h1>
</header>
`;
